extensions/dacpac/src/wizard/pages/extractConfigPage.ts

```diff
@@ -108,16 +108,17 @@ export class ExtractConfigPage extends DacFxConfigPage {
 
 	private async createVersionTextBox(): Promise<azdata.FormComponent> {
 		this.versionTextBox = this.view.modelBuilder.inputBox().withProperties({
-			required: true
+			required: true,
+			ariaLabel: localize('dacFxExtract.versionTextBoxAriaLabel', 'Version')
 		}).component();
 
 		// default version
 		this.versionTextBox.value = '1.0.0.0';
-		this.versionTextBox.ariaLabel = localize('dacfx.versionAriaLabel', "Version");
 		this.model.version = this.versionTextBox.value;
 
 		this.versionTextBox.onTextChanged(async () => {
-			this.model.version = this.versionTextBox.value;
+			const versionInputText = this.versionTextBox.value;
```

Comment:
why create this const?

Reply:
This used to serve a purpose but no longer does. I'll remove this.